packages/bento-design-system/tsup.config.ts

```diff
@@ -2,7 +2,7 @@ import { defineConfig } from "tsup";
 import { vanillaExtractPlugin } from "@vanilla-extract/esbuild-plugin";
 
 export default defineConfig({
-  entry: ["src/index.ts", "src/vars.css.ts"],
+  entry: ["src/index.ts"],
```

Comment:
why are we building `vars.css.ts` as a separate file? In this way we're producing a `vars.css.js`, that will be processed also by the final project. The library is already producing a `.css` file with the vars, and in the final project we just need the TS vars, that can be re-exported from the index (so that they will be included in the `index.js` and will not be re-processed by the project's vanilla-extract plugin)